api-tests/eCommerce-helpdesk.api.tests.dev.json

```diff
@@ -213,6 +213,216 @@
 			},
 			"response": []
 		},
+
+		{
+			"name": "eCommerce SearchTransaction (v2) TRANSACTION_ID",
```

Comment:
what about move those api tests into a separate v2 postman collection? (maybe create a v1 and v2 subfolder also here as done per api specs)

Reply:
done here [commit](https://github.com/pagopa/pagopa-ecommerce-helpdesk-service/pull/118/commits/7f8d4b16b0ff2c7cb27b64728c64fcace8ec9ade)